nextflow.config

```diff
@@ -0,0 +1,132 @@
+/*
+ * -------------------------------------------------
+ *  nf-core/rnafusion Nextflow config file
+ * -------------------------------------------------
+ * Default config options for all environments.
+ * Cluster-specific config options should be saved
+ * in the conf folder and imported under a profile
+ * name here.
+ */
+
+// Global default params, used in configs
+params {
+  container = 'nfcore/rnafusion:1.0' // Container slug. Stable releases should specify release tag!
+  container_version = '1.0'
+  custom_config_version = 'master'
+  igenomesIgnore = false
+  test = false
+  
+  // Pipeline params
+  star_fusion = false
+  fusioncatcher = false
+  fc_extra_options = ''
+  fusion_inspector = false
+  ericscript = false
+  pizzly = false
+  pizzly_k = 31
+  squid = false
+
+  // Defaults
+  help = false
+  reads = "data/*{1,2}.fastq.gz"
+  singleEnd = false
+  outdir = './results'
+  igenomes_base = "./iGenomes"
+  tracedir = "${params.outdir}/pipeline_info"
+  clusterOptions = false
+  awsqueue = false
+  awsregion = 'eu-west-1'
+  readPaths = null
+}
+
+// Load base.config by default for all pipelines
+includeConfig 'conf/base.config'
+
+// Load nf-core custom profiles from different Institutions
+includeConfig "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}/nfcore_custom.config"
+
+profiles {
+  awsbatch { includeConfig 'conf/awsbatch.config' }
+  conda { process.conda = "$baseDir/environment.yml" }
+  docker { docker.enabled = true }
+  singularity { singularity.enabled = true }
+  test {
+    includeConfig 'conf/test.config'
+  }
+  munin_docker {
+    includeConfig 'conf/munin-docker.config'
+  }
+  munin_singularity {
+    includeConfig 'conf/munin-singularity.config'
+  }
+  uppmax_devel {
+    includeConfig 'conf/uppmax-devel.config'
+  }
+}
+
+// Load igenomes.config if required
+if(!params.igenomesIgnore){
+  includeConfig 'conf/igenomes.config'
+}
+
+// Capture exit codes from upstream processes when piping
+process.shell = ['/bin/bash', '-euo', 'pipefail']
+
+timeline {
+  enabled = true
+  file = "${params.tracedir}/pipeline_info/nf-core/rnafusion_timeline.html"
+}
+report {
+  enabled = true
+  file = "${params.tracedir}/pipeline_info/nf-core/rnafusion_report.html"
+}
+trace {
+  enabled = true
+  file = "${params.tracedir}/pipeline_info/nf-core/rnafusion_trace.txt"
+}
+dag {
+  enabled = true
+  file = "${params.tracedir}/pipeline_info/nf-core/rnafusion_dag.svg"
+}
+
+manifest {
+  name = 'nf-core/rnafusion'
+  description = 'Attempt at setting up a pipeline for fusion detection as an addon to the previously existing NGI-RNAseq pipeline.'
```

Comment:
Needs updating.

Reply:
Updated to: `Nextflow rnafusion analysis pipeline, part of the nf-core community.`